katello-certs-gen-rpm

```diff
@@ -340,8 +340,6 @@ RPMOPTS="--define \"_topdir $RPM_BUILD_DIR\"\
  --define '_specdir     %{_topdir}'\
  --define '_rpmdir      %{_topdir}'\
  --define '_srcrpmdir   %{_topdir}'\
- --define '_source_filedigest_algorithm md5'\
- --define '_binary_filedigest_algorithm md5'\
```

Comment:
this should have been removed a long time ago (when we dropped EL5 client support)